product/roundhouse.console/app.config

```diff
@@ -1,5 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
+  <system.data>
+    <DbProviderFactories>
+      <add name="Npgsql" invariant="Npgsql"
+      description=".Net Framework Data Provider for Postgres"
+      type="Npgsql.NpgsqlFactory, Npgsql, Version=3.0.3.0, Culture=neutral, PublicKeyToken=5d8b90d52f46fda7"/>
+    </DbProviderFactories>
+  </system.data>

```

Comment:
Would it be possible to put it in the class itself? If not totally understand. I think there was a reason I left it out of the config itself. I think the purpose was to try to be as agnostic as possible in the console/msbuild/lib.